debian/rules

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/make -f
+
+export DH_VERBOSE = 1
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+override_dh_install:
+	install -d usr/lib
```

Comment:
isn't all that covered by the CMake?

Reply:
Indeed CMake is used for the build. This `rules` files delegates the build to CMake: 
https://github.com/intel/safestringlib/pull/48/files#diff-889c5495b68d2a0c0d1f238818716a71939918499efbc0fc6d2994319624fc67R19

The install override ensures compatibility in a Debian environment with `DEB_HOST_MULTIARCH`.